packages/asc-ui/src/components/Link/LinkStyle.ts

```diff
@@ -40,6 +40,7 @@ export const InlineLinkStyleCSS = css`
 `
 export const DefaultLinkStyleCSS = css<Props>`
   display: inline-flex;
+  align-items: center;
```

Comment:
Perhaps add a case on line 78? `case 'with-chevron': css'align-items: center;'` with this style added, so it won't effect other links